tests/types/array.test.js

```diff
@@ -59,6 +59,23 @@ describe("ArrayType", function() {
       });
     });
 
+    describe("sort", () => {
+      let sorted;
+
+      beforeEach(() => {
+        let unsorted = create([String], ["c", "b", "a"]);
+        sorted = unsorted.sort();
+      });
+
+      it("has state", () => {
```

Comment:
Let's make this a bit more descriptive:

"It has a value that is the sorted version of the original value"